docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md

```diff
@@ -65,13 +65,13 @@ $ cd /opt/downloads
 Download the `replicated.tar.gz` file by executing the following command:
 
 ```sh
-$ wget https://downloads.yugabyte.com/replicated.tar.gz
+$ wget --trust-server-names https://get.replicated.com/airgap
 ```
 
 Download the `yugaware` binary and change the following number, as required:
 
 ```sh
-$ wget https://downloads.yugabyte.com/yugaware-2.1.2.0-b10.airgap
+$ wget https://downloads.yugabyte.com/yugaware-2.4.4.0-b7.airgap
```

Comment:
Seems odd to be providing a `stable` release download in the `latest` docs. Should it be this instead?
```suggestion
$ wget https://downloads.yugabyte.com/yugaware-2.7.1.1-b1.airgap
```

Reply:
(And if so, you'll need to also update the same file in the stable folder)

Reply:
fixed